manifests/generic-device-plugin.yaml

```diff
@@ -24,27 +24,63 @@ spec:
       - image: squat/generic-device-plugin
         args:
         - --device
-        - '{"name": "serial", "groups": [
-            {"paths": [{"path": "/dev/ttyUSB*"}]},
-            {"paths": [{"path": "/dev/ttyACM*"}]},
-            {"paths": [{"path": "/dev/tty.usb*"}]},
-            {"paths": [{"path": "/dev/cu.*"}]},
-            {"paths": [{"path": "/dev/cuaU*"}]},
-            {"paths": [{"path": "/dev/rfcomm*"}]}
-          ]}'
+        - |
+          name: serial
+          groups:
+            - paths:
+                - path: /dev/ttyUSB*
+            - paths:
+                - path: /dev/ttyACM*
+            - paths:
+                - path: /dev/tty.usb*
+            - paths:
+                - path: /dev/cu.*
+            - paths:
+                - path: /dev/cuaU*
+            - paths:
+                - path: /dev/rfcomm*
         - --device
-        - '{"name": "video", "groups": [{"paths": [{"path": "/dev/video0"}]}]}'
+        - |
+          name: video
+          groups:
+            - paths:
+                - path: /dev/video0
         - --device
-        - '{"name": "fuse", "groups": [{"count": 10, "paths": [{"path": "/dev/fuse"}]}]}'
+        - |
+          name: fuse
+          groups:
+            - count: 10
+              paths:
+                - path: /dev/fuse
         - --device
-        - '{"name": "audio", "groups": [{"count": 10, "paths": [{"path": "/dev/snd"}]}]}'
+        - |
+          name: audio
+          groups:
+            - count: 10
+              paths:
+                - path: /dev/snd
         - --device
-        - '{"name": "capture", "groups": [
-            {"paths": [{"path": "/dev/snd/controlC0"}, {"path": "/dev/snd/pcmC0D0c"}]},
-            {"paths": [{"path": "/dev/snd/controlC1", "mountPath": "/dev/snd/controlC0"}, {"path": "/dev/snd/pcmC1D0c", "mountPath": "/dev/snd/pcmC0D0c"}]},
-            {"paths": [{"path": "/dev/snd/controlC2", "mountPath": "/dev/snd/controlC0"}, {"path": "/dev/snd/pcmC2D0c", "mountPath": "/dev/snd/pcmC0D0c"}]},
-            {"paths": [{"path": "/dev/snd/controlC3", "mountPath": "/dev/snd/controlC0"}, {"path": "/dev/snd/pcmC3D0c", "mountPath": "/dev/snd/pcmC0D0c"}]}
-          ]}'
+        - |
+          name: capture
+          groups:
+            - paths:
+                - path: /dev/snd/controlC0
+                - path: /dev/snd/pcmC0D0c
+            - paths:
+                - path: /dev/snd/controlC1
+                  mountPath: /dev/snd/controlC0
+                - path: /dev/snd/pcmC1D0c
+                  mountPath: /dev/snd/pcmC0D0c
+            - paths:
+                - path: /dev/snd/controlC2
+                  mountPath: /dev/snd/controlC0
+                - path: /dev/snd/pcmC2D0c
+                  mountPath: /dev/snd/pcmC0D0c
+            - paths:
+                - path: /dev/snd/controlC3
+                  mountPath: /dev/snd/controlC0
+                - path: /dev/snd/pcmC3D0c
+                  mountPath: /dev/snd/pcmC0D0c
```

Comment:
You hero :') plz tell me you used a json->YAML converter and didn't have to write this by hand

Reply:
Haha yeah I just edited the file with vim, selected the portions I wanted to convert, then ran `yq -P` 😂